tests/scale/test_upgrade_scale.py

```
@@ -0,0 +1,453 @@
+        commands=shlex.split("sudo cp ~/stress-ng /usr/share/nginx/html/"),
+    )
+    yield f"http://{vm_with_nginx_service_scope_module.custom_service.instance.spec.clusterIPs[0]}:{PORT_80}/stress-ng"
+
+
+@pytest.fixture(scope="module")
+def vms_for_upgrade_test(
+    request,
+    scale_unprivileged_client,
+    upgrade_scale_namespace,
+    created_data_source_for_scale,
+    created_post_copy_migration_policy_for_upgrade,
```

Ruff (0.14.8)

Unused function argument: `created_post_copy_migration_policy_for_upgrade`

(ARG001)